### ISSUE TITLE
Safari tries to load images from relative paths before they are rewritten 

### ISSUE THREAD

**Issue by jaufgang** [mohamedkomalo/angular-relative-paths-in-partial#3]
When loading a template with relative urls for image paths, Safari tries to load the images using the relative path, gets a 404, and then later loads the image again with the correctly rewritten path.  This results in a list of 404 errors in the dev console and unnecessary http requests.

The same problem does not occur in Chrome or Firefox (all testing done on Mac)

**Comment by mohamedkomalo** [comment]
I have a Maverick VM, will try to reproduce this on it. What version of safari and mac do you use ?

**Comment by jaufgang** [comment]
Safari 8.0.6 on OsX Yosemite 10.10.3

**Comment by jaufgang** [comment]
I have just discovered another aspect of this issue.

While performing an SEO check of a newly released version of a site, I discovered that Googlebot registers 404 errors for the same image links that were showing up as 404 errors in Safari dev console.  This was picked up using the "Fetch as Google" feature in Google Search Console (https://support.google.com/webmasters/answer/6066468)

Goolebot can successfuly render angular pages, and it does reload those images after the links get rewritten, so the rendered snapshot looks like it should.  However it still registered that there were multiple 404 errors experienced while loading the page.  

The issue as I originally reported it was pretty innocuous, but the that that it appears with Googlebot is of concern because it could potentially affect search ranking.  (broken links on pages can negatively affect search rank scoring)
